plugins/dell-dock/fu-dell-dock-status.c

```diff
@@ -156,7 +173,7 @@ fu_dell_dock_status_init(FuDellDockStatus *self)
 {
 	fu_device_add_protocol(FU_DEVICE(self), "com.dell.dock");
 	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UPDATABLE);
-	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_UNSIGNED_PAYLOAD);
+	fu_device_add_flag(FU_DEVICE(self), FWUPD_DEVICE_FLAG_SIGNED_PAYLOAD);
```

Comment:
the reason this was set as unsigned is because MST in WD19 wasn't signed.  I think instead what should be done is push the flag down into each device type and set it properly for all of them.  IE MST if on WD19/WD22 is unsigned, but if on K2 it's signed.  All other devices signed.